public/tendersmash.js

```diff
@@ -430,6 +430,11 @@ app.controller("mainController", function ($scope, $http, $sce, $q, $timeout, pr
     return promises;
   }
 
+  $scope.getOrderedLists = function() {
+    var sorted = _.sortBy($scope.lists, function(x) { return x.name; });
```

Comment:
yeah, I liked that they were together. area would be common, maybe custom sort so they always end up at the top would be nice?